flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/config/KubernetesOperatorConfigOptions.java

```diff
@@ -233,6 +233,15 @@ public class KubernetesOperatorConfigOptions {
                     .withDescription(
                             "Comma separated list of namespaces the operator monitors for custom resources.");
 
+    @Documentation.Section(SECTION_SYSTEM)
+    public static final ConfigOption<String> OPERATOR_LABEL_SELECTOR =
```

Comment:
Please run `mvn package -Pgenerate-docs -pl flink-kubernetes-docs -nsu -DskipTests` to generate the config option in the configuration document.